### PR TITLE
Test negative failover/relocate scenarios via ACM UI

### PR DESCRIPTION
**Pull request by am-agrawa**
This is a negative test scenario for Regional DR solution to ensure that the failover/relocate operations via ACM UI can not be performed to the cluster where the workloads are already running.

Tested on ODF 4.13.0-201.stable and ACM 2.8.0-DOWNSTREAM-2023-05-15-23-47-00

### CODE REVIEW

**Comment by am-agrawa** [comment]
ODF 4.13.0-201.stable
ACM 2.8.0-DOWNSTREAM-2023-05-15-23-47-00

Test logs- http://pastebin.test.redhat.com/1100526

Also tested existing relocate flow- http://pastebin.test.redhat.com/1100532
and existing failover flow- http://pastebin.test.redhat.com/1100536

File: tests/disaster-recovery/regional-dr/test_failover.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def test_failover(
         pass the yaml conf/ocsci/dr_ui.yaml to trigger it.
 
         """
-        if config.RUN.get("rdr_relocate_via_ui"):
+        if config.RUN.get("rdr_failover_via_ui"):
```
[the remark this answers is not in the document]
Comment (am-agrawa) [reply tests/disaster-recovery/regional-dr/test_failover.py:71]: Done

File: tests/disaster-recovery/regional-dr/test_negative_failover_relocate_ui.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                workload_to_move=f"{rdr_workload.workload_name}-1",
+                policy_name=rdr_workload.dr_policy_name,
+                failover_or_preferred_cluster=primary_cluster_name,
+                action=constants.ACTION_RELOCATE,
+                move_workloads_to_same_cluster=True,
+            )
+            if result:
+                logger.info("Test passed!")
+            else:
+                logger.info("Test failed!")
```
[the remark this answers is not in the document]
Comment (am-agrawa) [reply tests/disaster-recovery/regional-dr/test_negative_failover_relocate_ui.py]: Done

File: tests/disaster-recovery/regional-dr/test_negative_failover_relocate_ui.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                scheduling_interval=scheduling_interval,
+                workload_to_move=f"{rdr_workload.workload_name}-1",
+                policy_name=rdr_workload.dr_policy_name,
+                failover_or_preferred_cluster=primary_cluster_name,
+                move_workloads_to_same_cluster=True,
+            )
+            if result:
+                logger.info("Test passed!")
+            else:
+                logger.info("Test failed!")
```
[the remark this answers is not in the document]
Comment (am-agrawa) [reply tests/disaster-recovery/regional-dr/test_negative_failover_relocate_ui.py]: Done

File: tests/disaster-recovery/regional-dr/test_negative_failover_relocate_ui.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+    """
+
+    @pytest.mark.polarion_id("OCS-4746")
+    def test_failover_to_same_cluster(
+        self,
+        setup_acm_ui,
+        nodes_multicluster,
+        dr_workload,
+        node_restart_teardown,
```
[the remark this answers is not in the document]
Comment (am-agrawa) [reply tests/disaster-recovery/regional-dr/test_negative_failover_relocate_ui.py]: Done

File: ocs_ci/helpers/dr_helpers_ui.py [hunk 5]
```diff
@@ -239,43 +240,91 @@ def failover_relocate_ui(
...
-        assert acm_obj.wait_until_expected_text_is_found(
-            locator=acm_loc["peer-ready"],
-            expected_text=constants.PEER_READY,
-        ), f"Peer is not ready, can not initiate {action}"
+        # TODO: Commented below lines due to Regression BZ2208637
+        # log.info("Check peer readiness")
+        # assert acm_obj.wait_until_expected_text_is_found(
+        #     locator=acm_loc["peer-ready"],
+        #     expected_text=constants.PEER_READY,
+        # ), f"Peer is not ready, can not initiate {action}"
```
[the remark this answers is not in the document]
Comment (am-agrawa) [reply ocs_ci/helpers/dr_helpers_ui.py:312]: Done- https://github.com/red-hat-storage/ocs-ci/issues/7667

**Comment by am-agrawa** [comment]
@prsurve @keesturam @ebenahar Could you pls review/approve/merge?

**Review by keesturam — APPROVED** [review]
LGTM

**Pull request merged**